# Acquired IMU Data
Header header
float64 time_stamp
float64 quat_x
float64 quat_y
float64 quat_z
float64 quat_w
float64 gyro_x
float64 gyro_y
float64 gyro_z
float64 accel_x
float64 accel_y
float64 accel_z
